debian/rules

```diff
@@ -1,33 +1,70 @@
 #!/usr/bin/make -f
+#include /usr/share/dpkg/default.mk
+#include /usr/share/dh-php/pkg-pecl.mk
 
 DH_VERBOSE = 1
 
-DPKG_EXPORT_BUILDFLAGS = 1
-include /usr/share/dpkg/default.mk
+# already in pkg-pecl.mk
+#DPKG_EXPORT_BUILDFLAGS = 1
 
-export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+# already in pkg-pecl.mk
+#export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
-%:
-	dh $@ 
+# package name should be snuffleupagus, not php-snuffleupagus
+PECL_NAME_OVERRIDE := snuffleupagus
+DH_PHP_VERSIONS_OVERRIDE := $(shell /usr/sbin/phpquery -V)
+include /usr/share/dh-php/pkg-pecl.mk
```

Comment:
Shouldn't `include` be at the top of the file?

Reply:
https://salsa.debian.org/php-team/pecl/tideways does the same.

Compilation fails if I include pkg-pecl.mk at the top of the file : 
dpkg-source: info: using source format '3.0 (quilt)'
dpkg-source: info: building snuffleupagus using existing ./snuffleupagus_0.3.0.orig.tar.gz
dpkg-source: error: cannot represent change to build-7.2/.libs/snuffleupagus.la:
dpkg-source: error:   new version is symlink to ../snuffleupagus.la
dpkg-source: error:   old version is nonexistent
dpkg-source: error: cannot represent change to build-7.1/.libs/snuffleupagus.la:
dpkg-source: error:   new version is symlink to ../snuffleupagus.la
dpkg-source: error:   old version is nonexistent
dpkg-source: error: cannot represent change to build-7.0/.libs/snuffleupagus.la:
dpkg-source: error:   new version is symlink to ../snuffleupagus.la
dpkg-source: error:   old version is nonexistent
[...]

I don't know why this happens.